src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -286,6 +298,7 @@ static void mlpackMain()
 
       Log::Fatal << "Can't get responses from training data since it has less "
           << "than 2 rows." << endl;
+      return;
```

Comment:
Technically the `return` here is unnecessary since `Log::Fatal` will throw an exception, but I don't mind putting it in to satisfy the static code analysis checker.

Reply:
> Technically the `return` here is unnecessary since `Log::Fatal` will throw an exception

Agreed, I was worried about this part. After checking if the `checker` can detect `Log::Fatal` one more, I'll look into another solution to avoid unnecessary things.